.github/workflows/close-abandoned-issues.yaml

```diff
@@ -0,0 +1,62 @@
+name: Close Abandoned Issues
+
+on:
+  schedule:
+    - cron: "0 */24 * * *"
```

Comment:
Is it necessary to run it every hour? Isn't once a day enough?

Reply:
As @shub8968 confirmed, it should run every day at midnight.